profile.cc

```diff
@@ -1,5 +1,31 @@
 #include "profile.h"
 
-const double Profile::GetTime() {
-  return 0.0;
+const double Profile::GetTime(bool ignore) {
+  double delta_x = goal_.position - current_.position;
+  double discrim = (current_.velocity * current_.velocity) + (2 * constraints_.kMaxAcceleration * delta_x);
+  
+  if (ignore) {  
+    double full_send_time = (current_.velocity + sqrt(discrim)) / constraints_.kMaxAcceleration;
+  //  double full_send_time = delta_x / (.5 * (goal_.velocity + current_.velocity));
+  //  double full_send_time = (current_.velocity + sqrt(discrim)) / constraints_.kMaxAcceleration;
+    return full_send_time;
+ } else {
+   // first part of trapezoid
+   double accel_time = (constraints_.kMaxAcceleration - current_.velocity) / constraints_.kMaxAcceleration;
```

Comment:
`constraints_.kMaxAcceleration - current_.velocity` doesn't make any sense

Reply:
ah, my bad. It should have been ```goal_velocity - current_velocity```